spec/index.spec.ts

```diff
@@ -464,9 +462,7 @@ describe('axiosRetry(axios, { retries, onRetry })', () => {
         // eslint-disable-next-line no-unused-expressions
         expect(requestConfig).not.toBe(undefined);
```

Comment:
It looks like many of those `eslint-disable-next-line no-unused-expressions` can be removed in this file.